Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
```

Comment:
Never mind, "compile on save" is not needed to be added... It is on by default..